Examples/ByFeature/FunctionalInjection/ScenarioMetadataSteps.fs

```diff
@@ -0,0 +1,11 @@
+module ScenarioMetadataSteps
+
+open System
+open NUnit.Framework
+open TickSpec
+
+let [<Then>] ``the test is named "(.+)"`` (testName: string) (scenarioMetadata: ScenarioMetadata) =
```

Comment:
suggest `scenario`, `meta` or `metadata` for param name; fine with type name being explicit but it's a bit long here?

Reply:
Sounds good to me, updated.